src/pages/Experiments/TasksMenuBlock/TasksMenu/style.less

```diff
@@ -9,6 +9,15 @@
 
   padding-left: 24px !important;
 
+  .icon-url {
+    position: absolute;
+    left: 220px;
```

Comment:
Defining fixed position like this is a bad practice. Try to use the `margin-left` CSS property to align the icon on the right.